react-native-purchases-ui/README.md

```diff
@@ -15,6 +15,133 @@ Whether you are building a new app or already have millions of customers, you ca
 
 Sign up to [get started for free](https://app.revenuecat.com/signup).
 
+---
+
 ## React Native Purchases UI
 
-React Native Purchases UI is the client for the [RevenueCat](https://www.revenuecat.com/) subscription and purchase tracking system. It is an open source framework that provides a wrapper around `StoreKit`, `Google Play Billing` and the RevenueCat backend to make implementing in-app purchases in `React Native` easy.
+React Native Purchases UI are the UI components for the [RevenueCat](https://www.revenuecat.com/) subscription and purchase tracking system. It allows you to present paywalls in your React Native application, which you can further customize through the RevenueCat dashboard. 
+
+## Installation
+
+Expo supports in-app payments and is compatible with react-native-purchases-ui. You can install this package in an non-development build using the Expo Go app. In this case, `react-native-purchases-ui` will run in a Preview API mode, as it requires some native modules that are not available in Expo Go. All the listed APIs below are available but have no effect. 
```

Comment:
Perhaps, for consistency throughout the docs let's have react-native-purchases-ui with single inverted quotes as well
```suggestion
Expo supports in-app payments and is compatible with `react-native-purchases-ui`. You can install this package in an non-development build using the Expo Go app. In this case, `react-native-purchases-ui` will run in a Preview API mode, as it requires some native modules that are not available in Expo Go. All the listed APIs below are available but have no effect. 
```